aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -249,6 +254,81 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    fun verify_key_rotation_signature(scheme: vector<u8>, public_key_bytes: vector<u8>, signature: vector<u8>, challenge: RotationProofChallenge) : vector<u8> {
```

Comment:
scheme should just be u8

Reply:
Also, can you please take the `rotation_challenge` by reference, to avoid an unnecessary copy when you call this function twice in `rotate_authentication_key`?

Reply:
yep just updated! thanks for the review : )